### ISSUE TITLE
Can we stop accepting unpreservable pages?

### ISSUE THREAD

**Issue by non-descriptive** [rust-lang/this-week-in-rust#5836]
Once in a while I meet articles in TWiR which are either located on some self-hosted site way beyond some kind of ddos protection, which may ban access to me because of VPN or geo-IP resolution, or behind paywall (some Medium.com if I remember right was like that). Those articles might be good but likely gonna perish in a long run - which may lead to dangling links in TWiR. Usually, I just run "Save page" in web.archive.org for those links and read it as they are, but sometimes resources ban archivation which means if it goes down - it dies with all the info.
Event pages, videos and audios is understandably hard to preserve, but texts are pretty easy, so could we stop accepting posts which are not archivable? Maybe make little routine to save them before adding them to new issue. 
Most recent example: 
557: [Async Rust: The new billion-dollar mistake?](https://kerkour.com/rust-async-billion-dollar-mistake)

**Comment by oskgo** [comment]
Most pages will perish in the long run, and "this week in Rust" isn't an archive. It's a newsletter. Wouldn't requiring preservability harm that purpose?

What is your criteria for pages being "preservable"? What are examples of preservable blog posts? Can blogs do something to *become* preservable?

I agree with not accepting blogs that are behind a paywall, but I haven't seen that happen recently and the Readme explicitly lists things behind a pay- or register-wall as not wanted.

**Comment by non-descriptive** [comment]
> What is your criteria for pages being "preservable"? 

If you can shove it into https://web.archive.org/ save page and have at least textual snapshot, than it fits named category. Otherwise the article will not be accessible for many people, because of reasons I listed above.

**Comment by extrawurst** [comment]
does web.archive.org have an API for this?

**Comment by non-descriptive** [comment]
Yes, [they have](https://archive.org/developers/index-apis.html). Wikipedia bots certainly use it to archive some external references, but I'm not sure which one they use exactly.

**Comment by oskgo** [comment]
It seems like the example site explicitly forbids archiving their site, [check out the "reproduction" section at the bottom](https://kerkour.com/) and blocks it. I thought the problem was accidental when I made my first comment.

I really don't like this, but for integrity reasons rather than accessibility.

**Comment by non-descriptive** [comment]
> It seems like the example site explicitly forbids archiving their site
I can't look at any of it because 
a) IP from my country are banned in their setting of Cloudflare front
b) IP addresses related to VPNs/proxies I managed to use are also banned by their CF front

So the way I tried to proxy read it is either using google translate or using Internet archive. Good articles normally are preserved and even have several snapshots already. But that exact article bans either of them.

**Comment by bennyvasquez** [comment]
I think it's an acceptable thought to try to do this for accessibility reasons, but I don't think it's in the best interest of our readers at large to blanket prohibit articles from being submitted *only* because they might not be archivable or might not be around forever.

**Comment by non-descriptive** [comment]
Guess not
